spec/account_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'account'
+
+describe Account do
+  
+  let(:account) { Account.new([], 2000) }
+  let(:entry) { double(:entry) }
+  let(:second_entry) { double(:entry) }
+  before(:each) do
```

Comment:
This `before` block sets up one instance that's used by all the tests.  It's great to reuse setup code.  However, try to make it so the instance that a test uses has the minimal required setup done on it.